ISSUE_TEMPLATE.md

```diff
@@ -1,39 +1,60 @@
-Please try to adhere to the guidelines below as much as possible when
-submitting your issue.
-- Verify that your issue is not being currently addressed by other
-[issues](https://github.com/scikit-learn/scikit-learn/issues?q=)
-or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls?q=).
-- If your issue is a usage question or does not potentially require
-changes to the codebase to be solved, then
-[StackOverflow](http://stackoverflow.com/questions/tagged/scikit-learn)
-(using the`[scikit-learn]` tag) or our
-[mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general)
-may be a better place to bring it up. For more information, see
-[User Questions](http://scikit-learn.org/stable/support.html#user-questions).
-
-If you are submitting a bug issue:
-- Please include your operating system type and version number, as well
-as your Python, scikit-learn, numpy, and scipy versions. This information
-can be found by runnning the following code snippet:
+<!--
+If your issue is a usage question or does not potentially require
+changes to the codebase to be solved, submit it here instead:
+- StackOverflow with the scikit-learn tag: http://stackoverflow.com/questions/tagged/scikit-learn
+- Mailing List: https://lists.sourceforge.net/lists/listinfo/scikit-learn-general
+For more information, see User Questions: http://scikit-learn.org/stable/support.html#user-questions
+-->
+
+<!-- Instructions For Filing a Bug: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#filing-bugs -->
+
+Description
+-----------
+<!-- Example: Joblib Error thrown when calling fit on LatentDirichletAllocation with evaluate_every > 0-->
+
+Steps/Code to Reproduce
+-----------------------
+<!--
+Example:
+```
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.decomposition import LatentDirichletAllocation
+
+docs = ["Help I have a bug" for i in range(1000)]
+
+vectorizer = CountVectorizer(input=docs,analyzer='word')
+lda_features = vectorizer.fit_transform(docs)
+
+lda_model = LatentDirichletAllocation(
+    n_topics=10,
+    learning_method='online',
+    evaluate_every=10,
+    n_jobs=4,
+)
+model = lda_model.fit(lda_features)
 ```
+If the code is long, feel free to put it in a public gist:
+https://gist.github.com and link it.
```

Comment:
Work-around just rephrase the sentence: 
"If the code is too long, feel free to put it in a public gist and link it in the issue: https://gist.github.com"